### ISSUE TITLE
 Branding and redesign

### ISSUE THREAD

**Issue by Deveshshankar** [bithyve/hexa#6128]
Regarding broadcast Notification, I was thinking instead of giving notification in the notification panel.
We can give a direct pop-up in the app with 2 CTA (update or cancel) after that if a user Clicks on "Update" he will be redirected to the play store or app store.
 Reference image
https://bithyve-workspace.slack.com/archives/D03K564LPHU/p1663587743924419
@mide66 We can connect for discussion

**Comment by mide66** [comment]
There has to be a notification panel regardless. What if the user doesn’t want to attend to the notification at that point in time. Where would they be able to check it?

**Comment by mide66** [comment]
We can implement push notifications 
@Pawan2792 it's possible right?

**Comment by mide66** [comment]
We would implement this!

**Comment by Deveshshankar** [comment]
Yes, it is in backlog when we will do rebranding we can do this at that time.
